packages/online-playground/src/App.vue

```diff
@@ -90,12 +103,10 @@ if (!hash) {
 }
 
 // persist state
-watchEffect(() => {
-  const newHash = store
-    .serialize()
-    .replace(/^#/, useDevMode.value ? `#__DEV__` : `#`)
-  history.replaceState({}, '', newHash)
-})
+watchEffect(() => history.replaceState({}, '', store.serialize()))
+
+// production mode is enabled
+productionMode.value = import.meta.env.PROD
```

Comment:
We probably need to remove this. It's better to use the dev version by default (like in the vuejs playground)